tests/test-kubeadm-dind-cluster.sh

```diff
@@ -70,7 +70,7 @@ verify_deploy(){
 
 main(){
 
-    if [[ -z "$BLUEMIX_AUTH" ]]
+    if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]
```

Comment:
Why would we want to only run this test if a pull request is detected? The point of this test is so that we can have a kubernetes deployment in CI.